src/ZendeskApi.Client/Resources/UserIdentityResource.cs

```diff
@@ -28,9 +29,7 @@ public IResponse<UserIdentity> Post(UserIdentityRequest request)
 
         public IResponse<UserIdentity> Put(UserIdentityRequest request)
         {
-            var response = Post(request);
```

Comment:
The lines I posted above are working with the changes in this PR. No need to delete the user and recreate them, just update the specific email identity. One other change I made is set "verified" to false when updating the email.